software/docuseal.yml

```
@@ -0,0 +1,14 @@
+name: "Docuseal"
+website_url: "https://www.docuseal.co"
+source_code_url: "https://github.com/docusealco/docuseal"
+description: "Create, fill, and sign digital documents (alternative to DocuSign)."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Ruby
+  - Nodejs
+  - Docker
```

```suggestion
  - Docker
```

I can't find documentation on how to install from source, Docker seems to be the only supported installation method.